corehq/apps/export/models/new.py

```diff
@@ -148,13 +148,15 @@ def create_default_from_export_item(group_schema_path, item, app_ids_and_version
         """
 
         is_main_table = group_schema_path == MAIN_TABLE
+        is_advanced = isinstance(item, SystemExportItem) and item.is_advanced
```

Comment:
Yeah, it's not the most elegant thing. We do already sort of break the rule of not storing presentation on `ExportItem`s by including the tag field, which is almost always just for presentation (except for the `row` tag, which has meaning :(  ). We could possibly store the information about which are advanced or not in the `const.py` file, then do a lookup into that structure from this function, but that might be a bit too convoluted...